_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+---
+title: 'Bitcoin Optech Newsletter #113'
+permalink: /en/newsletters/2020/09/02/
+name: 2020-09-02-newsletter
+slug: 2020-09-02-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed change to the way LN commitment
+transactions are constructed, summarizes discussion of a default signet, and
+links to a proposal for standardizing temporarily trusted LN channels. Also
+included are our regular sections with recently transcribed talks and
+conversations and notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Witness asymmetric payment channels:** Lloyd Fournier [posted][witness asymmetric payment channels] to
+  the Lightning-Dev mailing list this week with a request for feedback
+  on a proposed change to the way LN commitment transactions are
+  created.  Currently, in-channel payments are made by constructing two
+  transactions that conflict with each other, meaning Bitcoin's
+  consensus rules will only allow one of the transactions to be included
+  in the block chain.  Each of these transactions includes the same two
+  outputs with the same two amounts---one paying Alice and one paying
+  Bob---but with slightly different scripts.
+
+  The script for Alice allows her to unilaterally close the channel, but
+  it gives Bob a chance to spend her funds if he knows a secret that
+  Alice previously created.  Similarly, if Bob unilaterally closes the
+  channel, Alice gets a chance to spend his funds using a secret he
+  created.  Giving your counterparty a secret that allows them to spend
+  your money makes it possible to effectively *revoke* that old channel
+  state so that you're strongly incentivized to publish the latest
+  unrevoked channel state onchain when it comes time to close the
+  channel.
+
+  What Fournier proposed this week was only creating a single commitment
+  transaction that would be shared between
+  Alice and Bob.  When creating the transaction, Alice creates two
+  private keys she'll need to spend her output and gives Bob the
+  corresponding public keys.  Bob then gives Alice a [signature
+  adaptor][topic adaptor signatures] for his half of the 2-of-2 multisig spend in the commitment
+  transaction; the adaptor can only be transformed into Bob's signature
+  if that signature will reveal one of Alice's private keys to Bob.  The
+  other private key Alice created is the revocation private key, which
+  she'll give to Bob if she wants to revoke this channel state in favor
+  of a later state.  Similarly, Bob generates his own keys and receives
+  a signature adaptor from Alice.
+
+  In this way, even though the commitment transaction is the same for
+  both parties, there will be a difference in the signatures (called
+  *witnesses*) each of them uses to unilaterally close the channel.
```

Comment:
On this point, I found this sentence by Fournier interesting:

    The guiding principle is to keep commitment transactions symmetric but to use
    the asymmetric knowledge of the parties to simulate the way lightning works
    right now.